databricks_cli/groups/cli.py

```diff
@@ -0,0 +1,181 @@
+@eat_exceptions
+@provide_api_client
+def add_group_to_parent_group_cli(api_client, group, parent):
+    """Add an existing user to an existing group."""
+    GroupsApi(api_client).add_group_to_parent_group(group, parent)
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Add an existing user to an existing group')
+@click.option('--user', '-u', required=True,
+              help="The name of the user to be added to the group.")
+@click.option('--group', '-g', required=True,
+              help="The name of the group to add the user to.")
+@profile_option
+@eat_exceptions
+@provide_api_client
+def add_user_to_group_cli(api_client, user, group):
+    """Add an existing user to an existing group."""
+    GroupsApi(api_client).add_user_to_group(user, group)
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Create one or more new groups.')
+@click.argument('name', required=True, nargs=-1)
+@profile_option
+@eat_exceptions
+@provide_api_client
+def create_cli(api_client, name):
+    """Create one or more new groups."""
+    for item in name:
+        GroupsApi(api_client).create_group(item)
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Get the members of a group.')
+@click.argument('group', required=True)
+@profile_option
+@eat_exceptions
+@provide_api_client
+def list_members_cli(api_client, group):
+    """Get the users and groups that are members of a specific group."""
+    content = GroupsApi(api_client).list_members(group)
+    click.echo(pretty_format(content))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='List the parent groups a group belongs to.')
+@click.argument('group', required=True)
+@profile_option
+@eat_exceptions
+@provide_api_client
+def list_group_parents_cli(api_client, group):
+    """List the parent groups a group belongs to."""
+    content = GroupsApi(api_client).list_group_parents(group)
+    click.echo(pretty_format(content))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='List existing groups.')
+@profile_option
+@eat_exceptions
+@provide_api_client
+def list_groups_cli(api_client):
+    """List existing groups."""
+    content = GroupsApi(api_client).list_groups()
+    click.echo(pretty_format(content))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='List the groups a user belongs to.')
+@click.argument('user', required=True)
+@profile_option
+@eat_exceptions
+@provide_api_client
+def list_user_groups_cli(api_client, user):
+    """List the groups a user belongs to."""
+    content = GroupsApi(api_client).list_user_groups(user)
+    click.echo(pretty_format(content))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Remove a group.')
+@click.argument('group', required=True, nargs=-1)
+@profile_option
+@eat_exceptions
+@provide_api_client
+def remove_group_cli(api_client, group):
```

Comment:
Let's make this consistent with the API too. https://docs.databricks.com/api/latest/groups.html#delete